src/main/java/com/github/seratch/jslack/common/http/SlackHttpClient.java

```diff
@@ -38,6 +38,10 @@ public Response postJsonPostRequest(String url, Object obj) throws IOException {
         return okHttpClient.newCall(request).execute();
     }
 
+    public Response delete(Request request) throws IOException {
```

Comment:
Just in case, I meant the implementation of this method is not reasonable enough for me because the method can issue not only DELETE method call but anything else.

Reply:
better now?

Reply:
As you can see, not yet.

Reply:
ah - sorry I see it - hang on

Reply:
now?

Reply:
thanks, LGTM 👍